packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -414,6 +436,22 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
           'If you set --shard-index you need to also set --total-shards.');
     }
 
+    final bool enableVmService = boolArg('enable-vmservice');
+    if (experimentalFasterTesting && enableVmService) {
+      globals.printStatus(
+        '--enable-vmservice was parsed but will be ignored. This option is not '
+        'compatible with --experimental-faster-testing.',
+      );
+    }
+
+    final bool ipv6 = boolArg('ipv6');
+    if (experimentalFasterTesting && enableVmService) {
+      globals.printStatus(
+        '--ipv6 was parsed but will be ignored. This option is not compatible '
+        'with --experimental-faster-testing.',
```

Comment:
Why not?

This seems like something that should work.

Reply:
This option is used to make the test harness server use IPv6, so I don't think it's applicable when using `--experimental-faster-testing` because `--experimental-faster-testing` doesn't use a test harness server.

Reply:
Ahh ok, consider adding a small explanation even if only in comments on this.

Reply:
I've added a comment.